test/Microsoft.SqlTools.ServiceLayer.Test.Common/TestData/ObjectExplorer/Baselines/AllSqlObjects.txt

```diff
@@ -0,0 +1,136 @@
+NodeType: Table Label: HumanResources.Employee
```

Comment:
I wonder if we should stop using AdventureWorks for testing, and perhaps create a simpler database schema with 1 single item for each object type (i.e. 1 table, 1 view, 1 sproc, etc.)...not related to this PR, but it may help with some of the test duration issues we're seeing.